src/i18n/es.js

```diff
@@ -194,15 +194,15 @@ export const messages_es = {
   'editor.image.brush_color': `Color del pincel`,
   'editor.image.brush_size': `Tamaño del pincel`,
   'editor.image.brush_type': `Tipo de pincel`,
-  'editor.image.canvas_background': `Fondo de lienzo`,
+  'editor.image.canvas_background': `Fondo del lienzo`,
   'editor.image.changed_background': `Modificado: fondo`,
   'editor.image.changed_background_image': `Modificado: imagen de fondo`,
   'editor.image.changed_style': `Modificado: estilo`,
   'editor.image.drop_or_upload': `Soltar o subir`,
   'editor.image.drop_upload': `Suelta una nueva imagen aquí, o haz clic para seleccionar los ficheros a subir.`,
   'editor.image.full_width_mobile': `Anchura total en dispositivos móviles`,
-  'editor.image.image_options': `Opciones Imagen`,
-  'editor.image.image_url': `URL Imagen`,
+  'editor.image.image_options': `Opciones de imagen`,
```

Comment:
Acá sería imagen en abstracto o la imagen que se ve actualmente?

Si es el segundo caso, creo que debería ser:

```suggestion
  'editor.image.image_options': `Opciones de la imagen`,
```

Reply:
No encontré donde usamos este texto, me suena que es el segundo caso.

Reply:
No pude comprobar donde se usa este texto, pienso que deberíamos usar los artículos como propones y si luego surge la necesidad de ajustar el texto nuevamente corregimos.

Reply:
🚀